src/app/technical-record/weights/weights.component.spec.ts

```diff
@@ -1,13 +1,37 @@
-import {async, ComponentFixture, getTestBed, TestBed} from '@angular/core/testing';
-import {TechRecordHelpersService} from '@app/technical-record/tech-record-helpers.service';
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
 import {SharedModule} from '@app/shared/shared.module';
 import {CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
 import { WeightsComponent } from './weights.component';
+import { TESTING_UTILS } from '../../utils/testing.utils';
+import { TechRecord } from '../../models/tech-record.model';
 
 describe('WeightsComponent', () => {
 
   let component: WeightsComponent;
   let fixture: ComponentFixture<WeightsComponent>;
+  const techRecordWeights: TechRecord = {
```

Comment:
Is it going to be the same shape across vehicleTypes?
I was wondering if this could be moved into the utils file we have for testing instead if this was the case?